src/Mono.Android/Java.Interop/TypeManager.cs

```diff
@@ -21,26 +21,6 @@ public static partial class TypeManager {
 
 		internal static IntPtr id_Class_getName;
 
-		static TypeManager ()
```

Comment:
*Relatedly*, *in this file*, we want to *remove* the allocations on lines 19  & 20, as that will require JIT'ing Dictionary instantiations which we'd rather avoid if at all possible, especially *on-device*, when we shouldn't really need those fields *at all*.  (Not needing them will require additional changes, but they shouldn't be needed...)

Reply:
Yep, that's the next step here.